### PR TITLE
Changes to Homepage

### PR DESCRIPTION
**Pull request by js-ts**
Site can be viewed at [site](https://cranky-lumiere-5df5c6.netlify.app/) 
Changes Made :
+ Added Favicon
+ Changed logo size
+ Added Top Banner
+ Added Banner
+ Added Points Section
+ Added Component Cards
+ Added Adopters Section
+ Changed controlplane.png
Will Add :
+ Blog
In the Next PR

### CODE REVIEW

**Review by theofpa — CHANGES_REQUESTED** [review]
Can we skip the files about building a docker image for mkdocs? It's better to keep the repo clean from the way of publishing it and focus on the content.

File: overrides/home.html [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+        <div>
+          <h2>Highly scalable and standards based
+            <br>
+            Model Inference Platform on Kubernetes
+            <br>
+
+            for Trusted AI
+          </h2>
+          <p style="display: block">
+            <a href="{{ 'getting_started/' | url }}" class="md-button md-button--primary">
```
Comment (yuzisun) [inline overrides/home.html:22]: /getting_started/get_started/gc

File: overrides/home.html [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+        <li class="task-list-item">
+          <input type="checkbox" checked disabled /><span class="task-list-indicator"></span>Provides <b>performant,
+            standardized inference protocol</b> across <b>ML frameworks</b>.
+        </li>
+        <li class="task-list-item">
+          <input type="checkbox" checked disabled /><span class="task-list-indicator"></span>Support modern
+          <b>serverless inference workload</b> with <b>Autoscaling</b> including <b>Scale to Zero on GPU</b>.
+        </li>
+        <li class="task-list-item">
+          <input type="checkbox" checked disabled /><span class="task-list-indicator"></span>Provides <b>high scalability</b>, <b>density packing</b> and <b>intelligent routing</b> using <b>ModelMesh</b>
```
Comment (yuzisun) [inline overrides/home.html:50]: nit: add period at the end.

**Review by yuzisun — APPROVED** [review]

**Comment by yuzisun** [comment]
Really awesome work! Thanks @js-ts !!

**Comment by js-ts** [comment]
@theofpa Deleted All the docker and deployment files

Deleted .dockerignore
Deleted .gitlab-ci.yml
Deleted Dockerfile
Deleted amplify.yml
Deleted docker-compose.yml

**Comment by theofpa** [comment]
Thank you @js-ts great stuff!

**Review by theofpa — APPROVED** [review]

**Pull request merged**